client/components/EntryFormRow/index.js

```diff
@@ -1,20 +1,57 @@
 import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { getCategories } from 'modules/categories';
+import { actions as AppActions } from 'modules/transactions';
 import DataSelector from './DataSelector';
 import styles from './style.scss';
 
+@connect(
```

Comment:
Yeah, totally. This container does more than just presentation and that's fine. Otherwise, we'd use `@connect` in App.js and then pass down the props all over the place, right? That wouldn't be very intuitive. Not just that, but if we transfer props down the chain too much, we end up having unnecessary re-renders.